lib/searchyll/indexer.rb

```diff
@@ -30,6 +31,10 @@ def initialize(configuration)
       self.working       = true
       self.timestamp     = Time.now
       self.batch_size    = BATCH_SIZE
+      # Mapping fields JSON file path
+      @@mapping_file_path = configuration.elasticsearch_mapping_fields
+      # Analysis fields JSON file path
+      @@analysis_file_path = configuration.elasticsearch_analysis_fields
```

Comment:
What's the intent behind making this a class method if you are going to adjust it at the time of instance construction? Could this not be `self. analysis_file_path` like the rest of the fields?

Reply:
@drusellers Sorry for the late answer! There's not a specified reason for which i've done so in that way. Maybe the only reason is that, not being a ruby developer, during developing I couldn't get self.analysis_file_path (and the other) to works. But now, in hindsight, i think that the problem wasn't relative to variables' scope... so maybe is the case to use self keyword.